assignments/array-methods.js

```diff
@@ -56,25 +56,58 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
-console.log(fullName);
+
+runners.forEach((runners) => {
+
+    fullName.push(runners.first_name + runners.last_name);
+});
+
+console.log(fullName); 
+
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
-console.log(allCaps); 
+
+ var upperFirstName = runners.map((runner) => {
+ var myvariable = runner.first_name.toUpperCase();
+ allCaps.push(myvariable);  
```

Comment:
map creates a new array in place. There's no need for the allCaps array on line 70 or to push to a new array.
```suggestion
const allCaps = runners.map(runner => {
  return runner.first_name.toUpperCase()
}) 
```